src/App.tsx

```diff
@@ -8,35 +12,41 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is&nbsp;Tab 1
-      </h1>
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
 
-      <div>
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active">
-              <a href="#tab-1">Tab 1</a>
-            </li>
+  const onTabSelected = (tab: Tab) => {
+    if (tab.id !== selectedTab.id) {
+      setSelectedTab(tab);
+    }
+  };
 
-            <li className="">
-              <a href="#tab-2">Tab 2</a>
-            </li>
+  return (
+    <div className="App">
+      <div className="section">
+        <h1 className="title">
+          {`Selected tab is ${selectedTab.title}`}
+        </h1>
 
-            <li className="">
-              <a href="#tab-3">Tab 3</a>
-            </li>
-          </ul>
+        <div>
+          <Tabs
+            data-cy="tab-content"
+            tabs={tabs}
+            selectedTabId={selectedTab}
+            onTabSelected={onTabSelected}
+          />
         </div>
 
-        <div className="block" data-cy="tab-content">
-          Some text 1
+        <div
+          className={classNames('block', {
+            blockOne: selectedTab.id === String('tab-1'),
+            blockTwo: selectedTab.id === String('tab-2'),
+            blockThree: selectedTab.id === String('tab-3'),
+          })}
+          data-cy="tab-content"
+        >
+          {`${selectedTab.content}`}
```

Comment:
move it back to `Tabs` component, also why do you need this weird styles, what if you have 1000 tabs? remove them
```suggestion
        <div
          className="block"
          data-cy="tab-content"
        >
          {`${selectedTab.content}`}
```

Reply:
Я розумію що на тисячу ніхто такого не буде робитьи))
А як керувати переміщенням тексту під табами? Підкинь ідею